app/templates/direct-award/end-search.html

```diff
@@ -0,0 +1,78 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Save your search - Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with
+    items = [
+      {
+        "link": url_for('main.index'),
+        "label": "Digital Marketplace"
+      },
+      {
+        "link": url_for('external.buyer_dashboard'),
+        "label": "Your Account"
+      },
+      {
+        "link": url_for('direct_award.saved_search_overview', framework_framework=framework.framework),
+        "label": "Saved Searches"
+      },
+      {
+        "link": url_for('direct_award.view_project', framework_framework=framework.framework, project_id=project.id),
+        "label": project.name
+      },
+      {
+        "label": "End search"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+<div class="end-search-page">
+  <header class="page-heading">
+    <h1>
+      End search
+    </h1>
+  </header>
+
+  <div class="dmspeak">
+    <h2 class="heading-xmedium">Before you end your search</h2>
+  </div>
+  <p>You should only continue when you have finished searching for cloud services</p>
+  <p>and are ready to start comparing the services you’ve found</p>
+
```

Comment:
There's an errant paragraph el here

Reply:
Good spot; we were missing a couple of div wrappers to get the proper two-thirds styling. Have added.